thrust/CODE_OF_CONDUCT.md

```diff
@@ -1,96 +0,0 @@
-# Code of Conduct
```

Comment:
I intentionally removed this file, because it serves no purpose. 

The Code of Conduct is only relevant when cloning the repo and for that we already have the identical CCCL CoC

Shout if you disagree